pkgs/by-name/to/toolong/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  python3Packages,
```

```suggestion
  python311Packages,
```
we probably want to use 3.11 until treesitter works with 3.12